tests/test_configure_feedstock.py

```diff
@@ -899,12 +900,12 @@ def test_conda_build_tools(config_yaml):
         fp.write(unmodified)
         fp.write("conda_build_tool: does-not-exist")
 
-    with pytest.raises(AssertionError):
+    with pytest.raises(configure_feedstock.ExceptionGroup):
```

Comment:
I wonder if we should raise a subclass of the ExceptionGroup for better specificity. If not, I prefer we import from the package directly here.

Reply:
> If not, I prefer we import from the package directly here.

I didn't do that because then we'd have to duplicate the logic of where ExceptionGroup comes from. i.e. check if it is a builtin or use the backport.